myia/frontends/pytorch.py

```diff
@@ -5,6 +5,7 @@
 from collections import OrderedDict
 
 import torch
+import torch.nn.functional
```

Comment:
you don't need to import torch.nn.functional

Reply:
Done!